cli/testdata/coder_login_--help.golden

```diff
@@ -10,6 +10,9 @@ OPTIONS:
           Specifies an email address to use if creating the first user for the
           deployment.
 
+      --first-user-full-name string, $CODER_FIRST_USER_FULL_NAME
```

Comment:
Nit: I think we should use "name" instead of "full name" since it is uncommon.

Reply:
I originally used `name` but @dannykopping suggested `full_name` instead to avoid confusion between `name` and `username`.

Reply:
I see.... 👍